### PR TITLE
support occlum font for java

### PR DESCRIPTION
**Pull request by superajun-wsj**

### CODE REVIEW

**Comment by guzongmin** [comment]
Can you please add a readme file to tell user what is the motivation of the demo and how to use it?
And we could add this demo into the CI test, which could help us keep this demo's health.

**Comment by superajun-wsj** [comment]
add READER.md file

File: demos/font/font_support_for_java/run_java_font_app.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SCRIPT_NAME=run_java_font_app_internal.sh
+
+./create_java_font_app.sh
+
+docker run -it --device /dev/isgx --network host --rm -v `pwd`:`pwd` -w `pwd` occlum/occlum:0.15.1-ubuntu18.04 `pwd`/$SCRIPT_NAME
```
Comment (guzongmin) [inline demos/font/font_support_for_java/run_java_font_app.sh:7]: We need to use the latest occlum build. The easy way is add this command in the readme file.

Comment (superajun-wsj) [reply demos/font/font_support_for_java/run_java_font_app.sh:7]: Done

File: demos/font/font_support_for_java/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### How to Run
+
+Step 1: Create and build demo, also collect font dependency occlum needs
+
+./create_java_font_app.sh
+
+Step 2: Create occlum instance and run the demo in occlum's latest docker image release
+
+docker run -it --device /dev/isgx --rm -v `pwd`:`pwd` -w `pwd` occlum/occlum:0.16.0-ubuntu18.04 `pwd`/run_java_font_app_internal.sh
```
Comment (guzongmin) [inline demos/font/font_support_for_java/README.md:33]: Can you please use occlum/occlum:[latest version]-ubuntu18.04 in the readme file?

Comment (guzongmin) [inline demos/font/font_support_for_java/README.md:33]:
```suggestion
docker run -it --device /dev/isgx --rm -v `pwd`:`pwd` -w `pwd` occlum/occlum:[latest version]-ubuntu18.04 `pwd`/run_java_font_app_internal.sh
```

Comment (superajun-wsj) [reply demos/font/font_support_for_java/README.md:33]: OK

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**